packages/editor-ui/src/Interface.ts

```diff
@@ -314,8 +314,29 @@ export interface IWorkflowDb {
 	versionId: string;
 	usedCredentials?: IUsedCredential[];
 	meta?: WorkflowMetadata;
+	resource?: 'workflow';
```

Comment:
is the backend returning this now? Why is it needed here?

At first I assumed it's needed for the resource list, but I see we have `resourceType` for that? 
What's the difference here?

Should this be at `WorkflowResourceDB` and required, instead of here and optional?

Reply:
Yes, this is in the back-end response now. But makes sense what you suggested